docs/config/lua/keyassignment/PromptInputLine.md

```diff
@@ -32,6 +35,7 @@ config.keys = {
     mods = 'CTRL|SHIFT',
     action = act.PromptInputLine {
       description = 'Enter new name for tab',
+      with_content = 'Tab: ',
```

Comment:
I think that the colon in the value make this look a bit like a prompt rather than an initial value, and since we now support setting a custom prompt, I think it might be clearer as an example to change this.

```suggestion
      initial_value = 'My Tab Name',
```